scvi/models/distributions.py

```diff
@@ -43,7 +43,8 @@ def _convert_counts_logits_to_mean_disp(total_count, logits):
         :return: the mean and inverse overdispersion of the NB distribution.
     """
     theta = total_count
-    mu = logits.exp() * theta
+    # mu = logits.exp() * theta
+    mu = logits + torch.log(theta)
```

Comment:
We don't use this parameterization during optimization so I'm not sure how this will fix any of the numerical issues.

Reply:
We probably should call it `log_mu` here rather than `mu` if we're going to starting storing it on the log scale. Could get confusing otherwise.